src/store/modules/ability.js

```diff
@@ -72,14 +72,22 @@ export const useAbilityStore = defineStore(
 					: `/index.php/apps/larpingapp/api/abilities/${this.abilityItem.id}`
```

Comment:
Het maakt eigenlijk niet zoveel uit met deze, maar het is wel betere code om de `abilityItem` dat je meegeeft te gebruiken